src/textual/message_pump.py

```diff
@@ -76,9 +76,11 @@ def __new__(
                     )
 
         # Look for reactives with public AND private compute methods.
+        prefix = "compute_"
+        prefix_len = len("compute_")
```

Comment:
So... you know what I'm going to mention here, right? ;-)

Reply:
I'm guessing:

```suggestion
        prefix_len = len(prefix)
```